src/main/java/com/pdffiller/client/dto/CustomLogoInfo.java

```diff
@@ -0,0 +1,73 @@
+package com.pdffiller.client.dto;
+
+
+public class CustomLogoInfo {
+  private Long id = null;
+  private DateResponse created_at = null;
+  private Integer width = null;
+  private Integer height = null;
+  private Long filesize = null;
+  private String logo_url = "https://www.pdffiller.com/img/logo-pdffiller.png";
+
+  public Long getId() {
+    return id;
+  }
+
+  public void setId(Long id) {
+    this.id = id;
+  }
+
+  public DateResponse getCreated_at() {
+    return created_at;
+  }
+
+  public void setCreated_at(DateResponse created_at) {
+    this.created_at = created_at;
+  }
+
+  public Integer getWidth() {
+    return width;
+  }
+
+  public void setWidth(Integer width) {
+    this.width = width;
+  }
+
+  public Integer getHeight() {
+    return height;
+  }
+
+  public void setHeight(Integer height) {
+    this.height = height;
+  }
+
+  public Long getFilesize() {
+    return filesize;
+  }
+
+  public void setFilesize(Long filesize) {
+    this.filesize = filesize;
+  }
+
+  public String getLogo_url() {
+    return logo_url;
+  }
+
+  public void setLogo_url(String logo_url) {
```

Comment:
We use either camel style of snake style, not both. In this project I stick to the camel style